src/__tests__/components/ShoppingCart/EmptyCart.test.tsx

```diff
@@ -6,7 +6,7 @@ describe("Empty cart components", () => {
   test("Should render all empty elements", () => {
     renderWithMockedProvider(<EmptyCart />);
     const checkOutButton = screen.getByRole("button", { name: "ReturnHomeBtn" });
-    const cartTitle = screen.getByText("CART");
+    const cartTitle = screen.getByText("Shopping Cart");
```

Comment:
we usually do it with a [snapshot](https://jestjs.io/docs/snapshot-testing) to test UI.

Reply:
Thanks Molly! Will dig into it later when the cart function is all completed :)